code/game/gamemodes/intercept_report.dm

```diff
@@ -54,4 +45,28 @@
 			text += "A dangerous Wizards' Federation individual by the name of [pick(GLOB.wizard_first)] [pick(GLOB.wizard_second)] has recently escaped confinement from an unlisted prison facility. This \
 			man is a dangerous mutant with the ability to alter himself and the world around him by what he and his leaders believe to be magic. If this man attempts an attack on your station, \
 			his execution is highly encouraged, as is the preservation of his body for later study."
+		if("Internal Affairs")
+			text += "Nanotrasen denies any accusations of placing internal affairs agents onboard your station to eliminate inconvenient employees.  Any further accusations against Centcom for such \
+			actions will be met with a conversation with an official internal affairs agent."
+		if("monkey")
+			text += "Reports of an ancient [pick("retrovirus", "flesh eating bacteria", "disease", "magical curse blamed on viruses", "bananna blight")] outbreak that turn humans into monkies has been \
+			reported in your quadrant.  Any such infections may be treated with bananna juice.  If an outbreak occurs, ensure the station is quarantined to prevent a largescale outbreak at Centcom."
+		if("meteor")
+			text += "[pick("Asteroids have", "Meteors have", "Large rocks have", "Stellar minerals have", "Space hail has", "Debris has")] been detected near your station, and a collision is possible, though unlikely.  Be prepared \
+			for largescale impacts and destruction.  Please note that the debris will prevent the escape shuttle from arriving quickly."
+		if("devil")
+			text += "Infernal creatures have been seen nearby offering great boons in exchange for souls.  This is considered theft against Nanotrasen, as all employment contracts contain a lien on the \
+			employee's soul.  If anyone sells their soul in error, contact an attorney to overrule the sale.  Be warned that if the devil purchases enough souls, a gateway to hell may open."
+		if("Devil Agents")
+			text += "Multiple soul merchants have been spotted in the quadrant, and appear to be competing over who can purchase the most souls.  Be advised that they are likely to manufacture \
+			emergencies to encourage employees to sell their souls. If anyone sells their soul in error, contact an attorney to overrule the sale."
+		if("abduction")
+			text += "Nearby spaceships report crewmembers having been kidnapped and tortured by mysterious grey humanoids, before being sent back.  Be advised that the kidnapped crewmembers behave \

```

Comment:
[pick("kidnapped", "abducted", "capture")] and [pick("tortured", "experimented on", "probed")]